package.json

```diff
@@ -50,7 +51,7 @@
     "lint:ts": "tslint --config tslint.json --project tsconfig.json \"**/*.ts?(x)\"",
     "prettier": "prettier \"**/*.{json,md,scss,yml}\"",
     "release": "lerna publish",
-    "test": "yarn && yarn lint:js && yarn lint:ts && yarn lint:other && node bin/testUpdated.js",
+    "test": "yarn && yarn lint:js && yarn lint:ts && yarn lint:other && ts-node bin/testUpdated.ts",
```

Comment:
Seems like the `.js` file was not removed. Is it still used?

Reply:
It's not used anymore. Will check why it got readded. Probably when I merged from master. 💡